### PR TITLE
Performance monitoring quickstart

### PR DESCRIPTION
**Pull request by nohe427**
This is the completed version of performance monitoring quickstart. The quickstart allows the user to generate a random string and then records how long that process took and reports it to the performance monitor within the Firebase Console. 

**Monochrome Icon**
Screenshot 2022-11-28 at 10 42 03 AM

**Regular Icon**
Screenshot 2022-11-28 at 10 42 24 AM

**App**
Screenshot_1669660961

### CODE REVIEW

File: performance/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Introduction
+------------
+
+- [Read more about Firebase Performance](https://firebase.google.com/docs/perf-mon/)
+
+Getting Started
+---------------
+
+- [Add Firebase to your Flutter Project](https://firebase.google.com/docs/perf-mon/flutter/get-started).
+- Run the sample on a device.
```
Comment (kroikie) [inline performance/README.md:16]: device or emulator/simulator?

File: performance/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+Firebase Performance Quickstart
+==============================
+
+The Firebase Performance Android Quickstart app demonstrates measuring the performance of
+activities in your application.
```
Comment (kroikie) [inline performance/README.md:5]: The description should go into a bit more detail:

- what is the quickstart is doing
- what exactly is perfmon measuring
- how can the dev verify that perfmon is working as expected

File: performance/README.md (hunk 1 above)
Comment (nohe427) [reply performance/README.md]: Updated, lmk what you think

File: performance/README.md (hunk 2 above)
Comment (nohe427) [reply performance/README.md]: Added some detail, lmk what you think

File: performance/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Introduction
+------------
+
+- [Read more about Firebase Performance](https://firebase.google.com/docs/perf-mon/)
+
+Getting Started
+---------------
+
+- [Add Firebase to your Flutter Project](https://firebase.google.com/docs/perf-mon/flutter/get-started).
+- Run the sample on a simulator, emulator, or real device.
```
[the remark this answers is not in the document]
Comment (nohe427) [reply performance/README.md:21]: Done. Let me know what you think

**Review by kroikie — APPROVED** [review]

**Pull request merged**